[{"exception": "Exception", "inner": [{"provider": {"package": "java.lang", "class": "Throwable", "method": "<init>", "filename": "Throwable", "line": ""}, "caller": {"package": "org.eclipse.core.runtime", "class": "CoreException", "method": "<init>", "filename": "CoreException", "line": "35"}}, {"provider": {"package": "org.eclipse.core.runtime", "class": "CoreException", "method": "<init>", "filename": "CoreException", "line": "35"}, "caller": {"package": "org.eclipse.ui.internal", "class": "PluginAction", "method": "runWithEvent", "filename": "PluginAction", "line": "263"}}, {"provider": {"package": "org.eclipse.ui.internal", "class": "PluginAction", "method": "run", "filename": "PluginAction", "line": "225"}, "caller": {"package": "org.eclipse.swt.widgets", "class": "TypedListener", "method": "handleEvent", "filename": "TypedListener", "line": ""}}], "outer_pre": [{"provider": {"package": "java.lang", "class": "Throwable", "method": "<init>", "filename": "Throwable", "line": ""}, "caller": {"package": "org.eclipse.core.runtime", "class": "CoreException", "method": "<init>", "filename": "CoreException", "line": "35"}}, {"provider": {"package": "org.eclipse.core.runtime", "class": "CoreException", "method": "<init>", "filename": "CoreException", "line": "35"}, "caller": {"package": "org.eclipse.ui.internal", "class": "PluginAction", "method": "runWithEvent", "filename": "PluginAction", "line": "263"}}], "outer_suf": [], "all": [{"provider": {"package": "java.lang", "class": "Throwable", "method": "<init>", "filename": "Throwable", "line": ""}, "caller": {"package": "org.eclipse.core.runtime", "class": "CoreException", "method": "<init>", "filename": "CoreException", "line": "35"}}]}]